src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1076,4 +1076,22 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="ColumnHeader_PackageLevel" xml:space="preserve">
     <value>Package Level</value>
   </data>
+  <data name="Label_PackageDetails" xml:space="preserve">
+    <value>Package Details</value>
+  </data>
+  <data name="Error_UnableToLoadReadme" xml:space="preserve">
+    <value>An error occurred while trying to load the README.</value>
```

Comment:
Not sure if we want to localize README too, maybe we should lock that string https://ceapex.visualstudio.com/CEINTL/_wiki/wikis/CEINTL.wiki/8680/Localizability-Commenting-Overview?anchor=commonly-used-functional-commenting-instruction-(locver-rule)

Reply:
There are a couple of strings that contain the word "Readme" which are being localized. It looks like in Spanish it's currently being localized to Léame, which seems right? Happy to lock the read README but if we do, we should do it everywhere.

Reply:
Maybe check with @lyndaidaii to align on localization strategy with nuget.org.

Reply:
Sorry for the late reply, for example this https://docs.github.com/es/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes, keeps README even in Spanish (there is one that is translated to Leame tho) but I'll ask Localization team what to do, we can do it later for all occurrences